eventkit_cloud/api/views.py

```diff
@@ -425,17 +424,27 @@ def partial_update(self, request, uid=None, *args, **kwargs):
 
            """
         payload = request.data
-        if not "published" in payload:
-            return Response([{'detail': _('missing published state parameter')}], status.HTTP_400_BAD_REQUEST)
+        key = None
```

Comment:
While our UI implementation will probably make requests per property there is no reason the API shouldn't support updating multiple values at once.

With only one property it can be ok to update it explicitly, however now it is becoming a little unwieldy with two (and maybe more properties later), and makes extending this code challenging.  I would look at either using the django rest framework to do its job here, http://www.django-rest-framework.org/api-guide/serializers/#partial-updates.... or if that proves less than straight forward you can try something like...
```

        response = {}
        for attribute, value in payload.iteritems():
            if hasattr(job, attribute):
                setattr(job, attribute, value)
                response[attribute] = value
```
Which might be a little more flexible for future updates.